### PR TITLE
Fix world constraint issues

### PR DESCRIPTION
**Pull request by thegrb93**
Will fix: #306 and fix: #187

### CODE REVIEW

**Comment by dvdvideo1234** [comment]
Neat.. I noticed that `gmsv_ad2filestream_win32` is not in the repository anymore. Is it still needed ot I can delete it from `Steam\steamapps\common\GarrysMod\garrysmod\lua\bin`?

**Comment by thegrb93** [comment]
Pretty sure its not needed

File: lua/advdupe2/sv_clipboard.lua [hunk 1]
```diff
@@ -1076,7 +1076,7 @@ function AdvDupe2.duplicator.Paste( Player, EntityList, ConstraintList, Position
 			undo.SetPlayer( Player )
 		undo.Finish()
 
-		//if(Tool)then AdvDupe2.FinishPasting(Player, true) end
+		--if(Tool)then AdvDupe2.FinishPasting(Player, true) end
```
Comment (thegrb93) [inline lua/advdupe2/sv_clipboard.lua:1079]: Remember to delete this line

**Comment by dvdvideo1234** [comment]
@thegrb93 Any progress on this.. Tell me if I can help somehow ..

**Comment by thegrb93** [comment]
No, I just returned from vacation and fixed some other things. Once I catch up on irl work and have time, I can continue this.

If you want to help, the bug is in this function https://github.com/wiremod/advdupe2/blob/8e27851b9cc9db438203d1ccc436a689aefd22b8/lua/advdupe2/sv_clipboard.lua#L455

Before two props are constrained, they are positioned/angled to their relative locations when the constraint was originally created and then constrained and then returned to their original position/angle. The problem is the world cannot be moved/angled so this fails in some cases.

**Comment by dvdvideo1234** [comment]
@thegrb93 Maybe this will get fixed when Rubat fixed the `SANE` big. I will test it when I get home.

**Comment by dvdvideo1234** [comment]
This problem appears to be fixed also on the `pre-release` branch. [Here is the duplication](https://github.com/wiremod/advdupe2/files/5396499/adv_dupe_wconstr.txt). Anchor entity is created successfully  and the duplication is loaded with no EOF errors.

**Comment by thegrb93** [comment]
Ok that's good. I still need to fix the world rotation issues for other constraints.

**Pull request closed without merging**